src/components/Hashcat/Hashcat.tsx

```diff
@@ -200,104 +195,75 @@ const Hashcat = () => {
-                                {...form.getInputProps("minPwdLen")}
-                            />
-                        </Grid.Col>
-                        <Grid.Col span={6}>
-                            <NumberInput
-                                label={"Maximum password length"}
-                                placeholder={"Recommend not bigger than 8"}
-                                required
-                                {...form.getInputProps("maxPwdLen")}
-                            />
-                        </Grid.Col>
-                    </Grid>
+                    <TextInput label="Password File Path" {...form.getInputProps("passwordFilePath")} required />
                 )}
                 {isCharset && (
-                    <TextInput
-                        label={"Mask Charsets, refer: https://hashcat.net/hashcat"}
-                        placeholder={"eg: ?d?d?d?d"}
-                        {...form.getInputProps("maskCharsets")}
-                    />
+                    <>
+                        <TextInput label="Mask Charset" placeholder="?l?d" {...form.getInputProps("maskCharsets")} />
+                        <Grid>
+                            <Grid.Col span={6}>
+                                <NumberInput
+                                    label="Minimum Password Length"
+                                    {...form.getInputProps("minPwdLen")}
+                                    required
+                                />
+                            </Grid.Col>
+                            <Grid.Col span={6}>
+                                <NumberInput
+                                    label="Maximum Password Length"
+                                    {...form.getInputProps("maxPwdLen")}
+                                    required
+                                />
+                            </Grid.Col>
+                        </Grid>
+                    </>
                 )}
                 <TextInput
-                    label={"Additional command, refer: https://hashcat.net/hashcat"}
-                    placeholder={"eg: --force"}
+                    label="Additional Commands"
+                    placeholder="e.g. --force"
                     {...form.getInputProps("additionalCommand")}
                 />
-                <Alert
-                    icon={<IconAlertCircle size={16} />}
-                    radius="md"
-                    children={
-                        "Please try with additional command '--force', otherwise you have to download OpenCL for hashcat. "
-                    }
-                ></Alert>
-                <Button type={"submit"}>Crack</Button>
-                {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
-                <ConsoleWrapper output={output} clearOutputCallback={clearOutput} />
             </Stack>
+
+            <Button type="submit" mt="md">
+                Scan
```

Comment:
The button should be Start {title}
```suggestion
               Start {title}
```